src/Api/appsettings.Development.json

```
@@ -1,4 +1,7 @@
 {
+  "Flags": {
+    "IsRateLimitBypassEnabled": true
```

```suggestion
    "IsRateLimitBypassEnabled": ["qa-app-1","qa-app-2"]
```

I think this kind of pattern would solve what Matt described and also solves QA's requirements.